src/main/resources/META-INF/rewrite/junit5.yml

```diff
@@ -96,6 +96,8 @@ recipeList:
   - org.openrewrite.java.testing.junit5.UpdateTestAnnotation
   - org.openrewrite.java.testing.junit5.ParameterizedRunnerToParameterized
   - org.openrewrite.java.testing.junit5.JUnitParamsRunnerToParameterized
+  - org.openrewrite.java.testing.junit5.MockitoJUnitToMockitoExtension
+  - org.openrewrite.java.testing.junit5.ExpectedExceptionToAssertThrows
```

Comment:
Added the ExpectedExceptionToAssertThrows, since it was missed in a previous PR.